.github/workflows/create_release.yml

```diff
@@ -1,13 +1,19 @@
-name: Release Android APK
+name: "Create release"
 
-on: workflow_dispatch
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
I would rather not do this; we should be able to fix the process to work exactly like in ItchySats; let's run this on the workflow_dispatch.

Reply:
That's the same as we do in itchysats

Reply:
exactly 🙂